nep-Y.mediawiki

```diff
@@ -0,0 +1,331 @@
+<pre>
+  NEP: <to be assigned>
```

Comment:
NEP: 24? Right after https://github.com/neo-project/proposals/pull/156.

Also need to update README.

Reply:
> NEP: 24? Right after #156.
> 
> Also need to update README.

Maybe 25, https://github.com/neo-project/proposals/pull/155 is ready for 24

Reply:
Done.